distribution/lib/Standard/Base/0.0.0-dev/src/Network/Email.enso

```diff
@@ -1,20 +1,22 @@
 import project.Data.Text.Text
 import project.Data.Vector.No_Wrap
 import project.Data.Vector.Vector
-import project.Errors.Common.Missing_Argument
 import project.Errors.Common.Dry_Run_Operation
+import project.Errors.Common.Missing_Argument
+import project.Errors.Illegal_Argument.Illegal_Argument
 import project.Metadata.Display
 import project.Metadata.Widget
+import project.Network.Email.Email
 import project.Network.Email.Email_Address.Email_Address
 import project.Network.Email.Email_Attachment.Email_Attachment
 import project.Network.Email.Email_Body.Email_Body
-import project.Network.Email.Email
 import project.Network.Email.Email_Provider.Email_Provider
 import project.Network.Email.Email_Send_Response.Send_Response
 import project.Runtime.Context
 import project.Warning.Warning
-
+from project.Error import all
```

Comment:
didn't we want to avoid `import all` in general? Unless this somehow cannot be avoided because of extension methods...